data/stopwords/stopwords-pt.txt

```diff
@@ -1,147 +1,611 @@
-último
+# http://www.linguateca.pt/chave/stopwords/
```

Comment:
The stopwords code doesn't filter out commented lines like this.

Can you remove this line (and add a note at the bottom of README.md if you want to add attribution)? 

Can you also remove the blank line below this line for the same reason?

Thanks!